SayTheirNamesTests/ServiceTests/DeepLinkTests.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  DeepLinkTests.swift
+//  SayTheirNames
+//
+//  Copyright (c) 2020 Say Their Names Team (https://github.com/Say-Their-Name)
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+import XCTest
+@testable import Say_Their_Names
+
+/// Unit Tests for the Deep Linking mechanism.
+/// The test cases rely on properly building a set of `DeepLink`s
+/// from the DeepLinkHandler's `handle(urls:)` method.
+///
+class DeepLinkTests: XCTestCase {
+
+    // The URLs that will be used to construct the deep links.
+    var dlAbout: URL!
+    var dlPeopleJaoP: URL!
+    var dlPetitions: URL!
+    var dlDonations: URL!
+    var dlPetitionScurlock: URL!
+    var dlDonationFloydG: URL!
+    
+    // The scheme and host that should be used.
+    let dlScheme = "stn"
+    let dlHost = "saytheirname.netlify.app"
+    
+    // The relative paths for deep link URLs
+    let dlPathAbout = "about"
+    let dlPathPeopleJaoP = "profile/joao-pedro"
+    let dlPathPetitions = "petitions"
+    let dlPathDonations = "donations"
+    let dlPathPetitionScurlock = "petitions/petition-for-james-scurlock"
+    let dlPathDonationFloydG = "donate/donate-to-george-floyd"
```

Comment:
For everything above this line... Is there a reason we aren't using the `DeepLinkSupport` enum?

Reply:
Not public. I can change that, but wasn't sure if it was by design or not.

Reply:
Yeah they can be public for unit testing.

Reply:
I added the file to the test target. Will change the code to use that and re-submit.

Reply:
Added `DeepLinkSupport` as requested. Good call 🙌 !
See 3183342fb3508adce5a86bb9d442e4ee8ceb48b1